src/content/docs/apm/agents/go-agent/installation/install-automation-new-relic-go.mdx

```diff
@@ -0,0 +1,99 @@
+
+In an effort to reduce that manual effort, the Go agent team created the Go easy instrumentation tool, which is currently in preview. This tool will do most of the work for you by suggesting changes to your source code that instrument your application with the New Relic Go agent.
+
+To get started, check out this four-minute video, or skip down to [How it works](#how-it-works):
+
+[![asciicast](https://asciinema.org/a/r0Il7o2eMiZaLKHIlew3IL2nx.svg)](https://asciinema.org/a/r0Il7o2eMiZaLKHIlew3IL2nx)
+
+## Preview notice [#preview-notice]
+
+This feature is currently provided as part of a product preview and is subject to our New Relic Experimental policies. Recommended code changes are suggestions only and should be subject to human review for accuracy, applicability, and appropriateness for your environment. This feature should only be used in non-critical, non-production environments that do not contain sensitive data.
+
+This project, its code, and the UX are under heavy development, so you should expect it to change. Please take this into consideration when participating in this preview. If you encounter any issues, please report them using [Github issues](https://github.com/newrelic/go-agent-assisted-instrumentation) and fill out as much of the issue template as you can so we can improve this tool.
+
+## How it works [#how-it-works]
+
+This tool will not interfere with your application's operation, and it doesn't make any changes to your code directly. Here's what happens:
+
+* It analyzes your source code and identifies opportunities to instrument it.
+* It suggests changes to your code that use the New Relic Go agent SDK to capture telemetry data.
+* You review those sugestions that are inserted in a `.diff` file and decide which changes to apply to your source code.
+
+As part of the analysis, this tool may invoke `go get` or other Go language toolchain commands which may modify your `go.mod` file, but not your actual source code.
+
+<Callout variant="important">
+  This tool can't detect if you already have New Relic instrumentation. Please only use this on applications without any instrumentation.
+</Callout>
+
+## What is instrumented? [#what-is-instrumented]
+
+The scope of what this tool can instrument in your application is limited to these actions:
+
+* Capturing errors in any function wrapped or traced by a transaction
+* Tracing locally defined functions that are invoked in the application's main() method with a transaction
+* Tracing async functions and function literals with an async segment
+* Wrapping HTTP handlers
+* Injecting distributed tracing into external traffic
+
+Only the following Go packages and libraries are currently supported:
+
+* standard library
+* net/http
+
+## Installation [#go-easy-install]
+
+Before you start the installation steps below, make sure you have a version of Go installed that is within the support window for the current [Go programming language lifecycle](https://endoflife.date/go).
```

Comment:
Alternate: Before starting the following installation steps, ensure that the installed Go version is within the [Go programming language lifecycle](https://endoflife.date/go) support window.